### PR TITLE
Added Github Profile

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains
- Added Github Profile for the community

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @vkkhare @ayush268 @souravrhythm

**Comment by ramesht007** [comment]
hey @Ayush7614 can you resolve this conflict?

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+### 🏂 Haven't tried NimbleEdge  yet?
+Here are some steps to learn and use the NimbleEdge .
+
+- [EnvisEdge](https://github.com/NimbleEdge/EnvisEdge) Deploy recommendation engines with Edge Computing.
+- [community.nimbleedge.ai](https://github.com/NimbleEdge/community.nimbleedge.ai) Community Website of NimbleEdge.
+- [Sentinels](https://github.com/NimbleEdge/Sentinels) Code related to any applications / bots serving our discord community.
+
+## Contact
+For reporting bugs, requesting features, and discussing changes, developers should use Github issues and discussions on these repositories. Using issues, you can [maintainers, contributors and other  members](nimbleedge.ai/discord) of the community. To connect with the community and learn more about the software and community, new users should bookmark or visit the [Community Website of NimbleEdge](https://www.nimbleedge.ai/).
```
Comment (ramesht007) [inline profile/README.md:24]: can you remove the link from here?
```
[maintainers, contributors and other members](nimbleedge.ai/discord)
```

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+We love to build ❤ our product in open source.
+ - At [NimbleEdge](https://www.nimbleedge.ai/), We offer Edge Computing as a service.
+ - Join us so we can make NimbleEdge more robust and more inclusive.
+
+### 👓 Appendix
+
+- [Documentation](https://docs.nimbleedge.ai/)
+- [About Us](https://www.nimbleedge.ai/about-us)
+- [Project](https://www.nimbleedge.ai/#project)
```
Comment (ramesht007) [inline profile/README.md:36]: remove this

**Review by ramesht007 — CHANGES_REQUESTED** [review]

**Comment by Ayush7614** [comment]
cc @ramesht007

File: profile/README.md [hunk 3]
```diff
@@ -34,3 +34,25 @@ We love to build ❤ our product in open source.
 - [Careers Opportunities](https://www.nimbleedge.ai/careers)
 - [Code of Conduct](https://github.com/NimbleEdge/EnvisEdge/blob/main/CODE_OF_CONDUCT.md)
 
+
+<h1 align="center">
+
+  <br>
+  <img src="https://github.com/ramesht007/.github-1/blob/main/assets/nimbleedge-dark-logo.png#gh-light-mode-only" alt="NimbleEdge"/ height="250" width="800">
```
Comment (ramesht007) [inline profile/README.md:41]: remove this from here and this to the in place of current image

**Review by ramesht007 — CHANGES_REQUESTED** [review]

**Comment by Ayush7614** [comment]
@ramesht007 Done Please review it and merge it

File: profile/README.md [hunk 4]
```diff
@@ -34,19 +41,6 @@ We love to build ❤ our product in open source.
...
-  <img src="https://github.com/ramesht007/.github-1/blob/main/assets/nimbleedge-dark-logo.png#gh-light-mode-only" alt="NimbleEdge"/ height="250" width="800">
-  <img src="https://github.com/ramesht007/.github-1/blob/main/assets/nimbledge-white-logo.png#gh-dark-mode-only" alt="NimbleEdge"/ height="250" width="800">
-  <br>
-  <br>
-  Join the Revolution Towards The Edge
-  <br>
-</h1>
-
-
 <h3 align="center">
```
Comment (ramesht007) [inline profile/README.md:44]: add this heading, to the top

**Review by ramesht007 — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 4 above)
Comment (Ayush7614) [reply profile/README.md]: Done @ramesht007 now you can review it

File: profile/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+# Welcome to NimbleEdge 👋
```
Comment (ramesht007) [inline profile/README.md:1]: Remove this

**Review by ramesht007 — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 5 above)
Comment (Ayush7614) [reply profile/README.md]: done cc @ramesht007

File: profile/README.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+<p align="center">
+<a href="nimbleedge.ai/discord" alt="discord"><img src="https://img.shields.io/badge/discord-@nimbleedge--discord-green.svg" alt="discord"></img></a>
+<a href="https://github.com/NimbleEdge" alt="github"><img src="https://img.shields.io/badge/github-@nimbleedge--github-orange.svg" alt="github"></img></a>
+<a href="https://twitter.com/NimbleedgeINC" alt="twitter"><img src="https://img.shields.io/badge/twitter-@nimbleedge--twitter-yellow.svg" alt="twitter"></img></a>
+<a href="https://www.linkedin.com/company/nimbleedge/" alt="linkedin"><img src="https://img.shields.io/badge/linkedin-@nimbleedge--linkedin-red.svg" alt="linkedin"></img></a>
+
+</p>
+
+[NimbleEdge](https://www.nimbleedge.ai/) NimbleEdge is leading the movement towards real edge computing. Creating a challenging new age technology altering the status quo between the cloud and the users’ devices, a technology that is inherently privacy preserving.
```
Comment (ramesht007) [inline profile/README.md:30]: Remove `real` from this line

**Review by ramesht007 — CHANGES_REQUESTED** [review]

File: profile/README.md [hunk 7]
```diff
@@ -10,6 +10,16 @@
   <br>
 </h1>
 
+<h3 align="center">
+  <br>
+  OUR MISSION & VISION
```
Comment (ramesht007) [inline profile/README.md:14]: add this below banners

**Review by ramesht007 — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 7 above)
Comment (Ayush7614) [reply profile/README.md:12]: done cc @ramesht007

Comment (ramesht007) [reply profile/README.md:12]: it is not fixed yet, can try and remove this whole mission and vision part?

Comment (Ayush7614) [reply profile/README.md]: done cc @ramesht007

File: profile/README.md [hunk 8]
```diff
@@ -10,6 +10,16 @@
...
+  <br>
+</h3>
+<h4 align="center">
```
Comment (ramesht007) [inline profile/README.md:12]: add this below discord banner and other banners

**Review by ramesht007 — CHANGES_REQUESTED** [review]

**Comment by ramesht007** [comment]
any updates @Ayush7614 ??

File: profile/README.md (hunk 8 above)
Comment (Ayush7614) [reply profile/README.md:12]: image

**Comment by Ayush7614** [comment]
> any updates @Ayush7614 ??

It looks not good can i remove vision and mission heading

**Comment by Ayush7614** [comment]
image
If i added only after discord banner it also not looks good

**Comment by ramesht007** [comment]
remove these two

Screenshot 2022-05-09 at 11 35 48 AM

**Comment by Ayush7614** [comment]
image

**Comment by Ayush7614** [comment]
this one also

**Comment by ramesht007** [comment]
> this one also

just remove the MISSION and VISION heading and the text above banner.

**Comment by Ayush7614** [comment]
> remove these two
> 
> Screenshot 2022-05-09 at 11 35 48 AM

Done

**Comment by Ayush7614** [comment]
> > this one also
> 
> just remove the MISSION and VISION heading and the text above banner.

Done cc @ramesht007

**Comment by ramesht007** [comment]
> > > this one also
> > 
> > 
> > just remove the MISSION and VISION heading and the text above banner.
> 
> Done cc @ramesht007

Perfect!!

**Review by ramesht007 — APPROVED** [review]
LGTM!!

**Pull request merged**